Makefile.v8

```diff
@@ -22,21 +26,35 @@ plv8_config.h: $(AUTOV8_LIB)
 
 $(AUTOV8_DIR):
 	mkdir -p build
-	curl -L 'https://github.com/v8/v8/archive/$(AUTOV8_VERSION).tar.gz' \
-		| tar zxf - -C build
+	cd build; fetch v8; cd v8; git checkout $(AUTOV8_VERSION); gclient sync
```

Comment:
Using "make dependencies" in the v8 source tree instead of gclient leads to build errors (https://code.google.com/p/v8/issues/detail?id=3835#c11). Using gclient avoids these, and is currently the only way of building v8 which is supported by the v8 project.